twitteranalyser/settings.py

```diff
@@ -23,13 +23,14 @@
 OH_BASE_URL = 'https://www.openhumans.org'
 # OH_BASE_URL = 'https://staging.openhumans.org'
 APP_BASE_URL = os.getenv('APP_BASE_URL', 'http://127.0.0.1:5000/users')
-
+HEROKUCONFIG_APP_NAME = os.getenv('HEROKUCONFIG_APP_NAME', '')
+ON_HEROKU = bool(HEROKUCONFIG_APP_NAME)
```

Comment:
If you introduce these changes you will have to update the documentation as well. because so far the repository just relies on postgres iirc and there is no reference made to the `HEROKUCONFIG_APP_NAME` or `ON_HEROKU`. 

I would be fine with adding a simple `OH_HEROKU` parameter in the `settings.py` and then making reference to it in the `INSTALL.md`. The `herokuconfig_app_name` isn't really needed in this site.

Reply:
Sure, that makes absolute sense. Documentation needs to be updated. 
I'll make the requested changes ASAP.